.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What does this file?

Reply:
This file is to debug in real time and run the python script in VSCode.

<img width="480" alt="Screenshot 2024-01-16 at 18 40 34" src="https://github.com/natacardona/learning_python_with_me/assets/7727385/07ea1f37-a7b0-4131-9f03-ea97b3438a58">

Reply:
ok,thanks